.circleci/config.yml

```diff
@@ -22,23 +19,25 @@ precheck_steps: &precheck_steps
 # Build crates for embedded target
 target_steps: &target_steps
   docker:
-    - image: circleci/rust:1.40.0
+    - image: jamwaffles/circleci-embedded-graphics:1.40.0
   steps:
     - checkout
     - restore_cache:
         keys:
           - v5-embedded-graphics-{{ .Environment.CIRCLE_JOB }}-{{ checksum "embedded-graphics/Cargo.toml" }}-{{ checksum "tinybmp/Cargo.toml" }}-{{ checksum "tinytga/Cargo.toml" }}-{{ checksum "simulator/Cargo.toml" }}
     - run: |
         SYSROOT=$(rustc --print sysroot)
-
-        if [[ ! "$SYSROOT" =~ "$TARGET" ]]; then
-          rustup target add $TARGET
-        else
-          echo "Target $TARGET is already installed"
-        fi
-    - run: cargo install just
+        TARGETS=("arm-unknown-linux-gnueabi" "armv7-unknown-linux-gnueabihf" "x86_64-unknown-linux-gnu" "x86_64-unknown-linux-musl" "thumbv6m-none-eabi" "thumbv7em-none-eabi" "thumbv7em-none-eabihf" "thumbv7m-none-eabi")
+
+        for TARGET in "${TARGETS[@]}"; do
+          if [[ ! "$SYSROOT" =~ "$TARGET" ]]; then
+            rustup target add $TARGET
+          else
+            echo "Target $TARGET is already installed"
+          fi
+        done
```

Comment:
Should we move this to the `justfile`? This way we wouldn't need to maintain two target lists.

Reply:
I did consider that to reduce the duplication. I decided not to originally because I thought it would be strange having a command that installs a bunch of targets, but then again we do have the command to _build_ those targets in the justfile already, so I'll move this script into it too.

Reply:
We could also do the opposite and call `just build-target $TARGET` in a loop from `config.yml`.

Reply:
We could, but I think I prefer the symmetry of `just install-targets` and `just build-targets` so I went with that.